modules/apps/app-builder/app-builder-test/src/testFunctional/macros/TableViewBuilder.macro

```diff
@@ -15,6 +25,8 @@ definition {
 
 	macro setName {
 		AppBuilderAdmin.setName(title = "${tableViewName}");
+
+		TableViewBuilder.save();
```

Comment:
Hey Brooke, I know that adding save action here we can increase the reuse (avoiding having this action in different places), but I would prefer to keep save action out of the setName macro because we will not be generating a dependency where setName is additionally doing a save action (and closing the TableViewBuilder in this case). 

If we decide to keep save action inside of setName macro, it will be obligated to be the last action of the creation of a Table View since it will close the TableViewBuilder. If we decide to keep save action out of setName macro, we can use setName macro any time during the creation of a Table View.